src/docgen/render/markdown.ts

```diff
@@ -233,3 +220,46 @@ export class Markdown {
     return line.replace(/\|/g, '\\|');
   }
 }
+
+export const defaultAnchorFormatter = (type: JsiiEntity) => {
+  return type.id;
+};
+
+export const defaultLinkFormatter = (type: JsiiEntity) => {
+  const name = type.fqn.split('.').pop()!;
+  return `<a href="#${type.id}">${name}</a>`;
+};
+
+function isJsiiType(value: any): value is JsiiEntity {
+  return typeof value === 'object' && value.fqn && value.id;
```

Comment:
This may explode if `value` is ever `null`. Thanks, JavaScript!

Reply:
`typeof null === 'object'`... TIL!